app/utils/TransitHolderAnswerHelper.scala

```diff
@@ -0,0 +1,111 @@
+  departureId: String,
+  cyaRefDataService: CheckYourAnswersReferenceDataService,
+  mode: Mode
+)(implicit messages: Messages, ec: ExecutionContext, hc: HeaderCarrier)
+    extends AnswersHelper(userAnswers, departureId, mode) {
+
+  def eoriYesNoRow: SummaryListRow =
+    buildRowWithNoChangeLink(
+      prefix = "transitHolder.eoriYesNo",
+      answer = formatAsYesOrNo(userAnswers.departureData.HolderOfTheTransitProcedure.identificationNumber.isDefined)
+    )
+
+  def eoriRow: Option[SummaryListRow] = userAnswers.departureData.HolderOfTheTransitProcedure.identificationNumber.map {
+    eori =>
+      buildRowWithNoChangeLink(
+        prefix = "transitHolder.eori",
+        answer = formatAsText(eori)
+      )
+  }
+
+  def nameRow: Option[SummaryListRow] = userAnswers.departureData.HolderOfTheTransitProcedure.ContactPerson.map {
+    person =>
+      buildRowWithNoChangeLink(
+        prefix = "transitHolder.name",
+        answer = formatAsText(person.name)
+      )
+  }
+
+  def countryRow: Option[Future[SummaryListRow]] = userAnswers.departureData.HolderOfTheTransitProcedure.Address.map {
+    address =>
+      cyaRefDataService
+        .getCountry(address.country)
+        .map(
+          country =>
+            buildRowWithNoChangeLink(
+              prefix = "transitHolder.country",
+              answer = formatAsCountry(country)
+            )
+        )
+  }
+
+  def addressRow: Option[SummaryListRow] = userAnswers.departureData.HolderOfTheTransitProcedure.Address.map {
+    address =>
+      buildRowWithNoChangeLink(
+        prefix = "transitHolder.address",
+        answer = formatAsDynamicAddress(address.toDynamicAddress)
+      )
+  }
+
+  def tirIdentificationRow: Option[SummaryListRow] = userAnswers.departureData.HolderOfTheTransitProcedure.TIRHolderIdentificationNumber.map {
+    tirIdentification =>
+      buildRowWithNoChangeLink(
+        prefix = "transitHolder.tirIdentification",
+        answer = formatAsText(tirIdentification)
+      )
+  }
+
+  def transitHolderSection: Future[Section] =
+    for {
+      eoriYesNoRow         <- Future.successful(Some(eoriYesNoRow))
+      eoriRow              <- eoriRow.toFuture
+      nameRow              <- nameRow.toFuture
+      countryRow           <- countryRow.toFuture
+      addressRow           <- addressRow.toFuture
+      tirIdentificationRow <- tirIdentificationRow.toFuture
+    } yield Section(
+      sectionTitle = messages("checkYourAnswers.transitHolder"),
+      Seq(eoriYesNoRow, eoriRow, nameRow, countryRow, addressRow, tirIdentificationRow).flatten
+    )
+
+  implicit class OptionImplicit(row: Option[SummaryListRow]) {
+    def toFuture: Future[Option[SummaryListRow]] = Future.successful(row)
+  }
+
+  implicit class OptionFutureImplicit(row: Option[Future[SummaryListRow]]) {
+
+    def toFuture: Future[Option[SummaryListRow]] = row match {
+      case Some(value) => value.map(Some(_))
+      case None        => Future.successful(None)
+    }
+  }
```

Comment:
These look like they could be useful in other answers helpers. Could they perhaps be moved to a package object inside utils?